scooby/report.py

```diff
@@ -283,6 +283,8 @@ def cols(html, version, name, ncol, i):
         html, i = cols(html, 'Architecture', self.architecture, self.ncol, i)
         if TOTAL_RAM:
             html, i = cols(html, 'RAM', self.total_ram, self.ncol, i)
+        else:
+            html, i = cols(html, '', '', 1, 0)
```

Comment:
I don't think we want this. Other entries can be added to that section by the user and this would add an awkward blank entry when the RAM isn't reported

Reply:
Now only Plain Text and HTML order are changed and nothing else.

Is this image good enough for the README ? Or is some data missing in compared to
the old image.

![image](https://user-images.githubusercontent.com/3840578/74086095-f8946c00-4a7f-11ea-950d-44b15def1f52.png)


--------------------------------------------------------------------------------
  Date: Sat Feb 08 14:30:36 2020 W. Europe Standard Time

                OS : Windows
            CPU(s) : 16
           Machine : AMD64
      Architecture : 64bit
               RAM : 63.9 GB
       Environment : Jupyter

  Python 3.7.3 (v3.7.3:ef4ec6ed12, Mar 25 2019, 22:22:05) [MSC v.1916 64 bit
  (AMD64)]

             numpy : 1.18.1
           IPython : 7.12.0
        matplotlib : 3.0.3
            scooby : 0.5.1
--------------------------------------------------------------------------------

Reply:
This is how the reports look now and I could include them to README

![Jupyter](https://user-images.githubusercontent.com/3840578/74131819-7b830700-4be5-11ea-89b2-6e2627b69db4.jpg)

```
--------------------------------------------------------------------------------
  Date: Mon Feb 10 09:00:45 2020 W. Europe Standard Time

                OS : Windows
            CPU(s) : 16
           Machine : AMD64
      Architecture : 64bit
               RAM : 31.9 GB
       Environment : IPython

  Python 3.7.6 | packaged by conda-forge | (default, Jan  7 2020, 21:48:41)
  [MSC v.1916 64 bit (AMD64)]

             numpy : 1.18.1
           IPython : 7.12.0
        matplotlib : 3.0.3
            scooby : 0.5.0

  Intel(R) Math Kernel Library Version 2019.0.4 Product Build 20190411 for
  Intel(R) 64 architecture applications
--------------------------------------------------------------------------------
```

```
--------------------------------------------------------------------------------
  Date: Mon Feb 10 09:00:56 2020 W. Europe Standard Time

                OS : Windows
            CPU(s) : 16
           Machine : AMD64
      Architecture : 64bit
               RAM : 31.9 GB
       Environment : IPython

  Python 3.7.6 | packaged by conda-forge | (default, Jan  7 2020, 21:48:41)
  [MSC v.1916 64 bit (AMD64)]

           pyvista : 0.23.1
               vtk : 8.1.2
        no_version : Could not import
    does_not_exist : Could not import
             numpy : 1.18.1
           IPython : 7.12.0
        matplotlib : 3.0.3
            scooby : 0.5.0

  Intel(R) Math Kernel Library Version 2019.0.4 Product Build 20190411 for
  Intel(R) 64 architecture applications
--------------------------------------------------------------------------------
```

Reply:
That looks good!